sys-devel/gettext/gettext-0.19.7.recipe

```diff
@@ -97,7 +107,6 @@ INSTALL()
 	make install AUTOCONF=: AUTOMAKE=: AUTOHEADER=: ACLOCAL=:
 
 	rm $libDir/charset.alias
-	rm $libDir/lib*.la
```

Comment:
@diversys: Thanks! objdump works like a charm!

@korli: **`grep ^dependency_libs /system/develop/lib/libintl.la`** yelds:
```
dependency_libs=' /packages/libiconv-1.13.1-6/.self/develop/lib/libiconv.la'
```
Should the INSTALL() in gettext install a modified libintl.la to have this?
```
dependency_libs=' /system/develop/lib/libiconv.la'
```
Would calling fixPkgconfig do the job?